crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__E274_E27.py.snap

```diff
@@ -9,10 +9,10 @@ expression: diagnostics
     fixable: false
   location:
     row: 28
-    column: 1
+    column: 2
```

Comment:
I checked these numbers and they match pycodestyle

```
crates/ruff/resources/test/fixtures/pycodestyle/E27.py:28:2: E274 tab before keyword
crates/ruff/resources/test/fixtures/pycodestyle/E27.py:30:5: E274 tab before keyword
```

The other tests are fixed (very hacky solution but the next PR replaces it anyway)